modules/ray_tracing/test/tests/trace_ray/adaptivity/adaptivity_1d.i

```diff
@@ -0,0 +1,96 @@
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = 2
+  []
+[]
+
+[Variables/u]
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
```

Comment:
diffusion problem is just here to trigger the adaptivity right>\?

Reply:
There needs to be a problem to solve. Solving a problem with just a line source is no bueno